### PR TITLE
Get shared guilds method

### PR DESCRIPTION
**Pull request by Vap0r1ze**

### CODE REVIEW

**Comment by GusCaplan** [comment]
quality pr

File: src/structures/User.js [hunk 1]
```diff
@@ -98,6 +98,15 @@ class User {
   }
 
   /**
+   * A collection of guilds that are shared bewteen the client and the user
```
Comment (meew0) [inline src/structures/User.js]: There's a typo here, should be "between"

**Comment by Gawdl3y** [comment]
Not sure this should be in the lib. For one, it will only work on users/members that are cached already.

**Comment by GusCaplan** [comment]
this functionality is implemented in #835 and resolves the caching issue mentioned by @Gawdl3y

**Pull request closed without merging**